### PR TITLE
Updated design of wesbite.

### PR DESCRIPTION
**Pull request by Trancever**
**Changed design of the home page of documentation website**

**This PR resolves issue #118 .**

Output from stylelint:
stylelint
 
Big screen:
big-screen

Mobile:
mobile1

Mobile:
mobile2

### CODE REVIEW

File: website/src/components/Header.js [hunk 1]
```diff
@@ -5,16 +5,22 @@ import { css } from 'linaria';
...
+const navHorizontalSpacing = 10;
 
 export default function Header() {
   return (
     <div className={header}>
       <Container className={headerContainer}>
-        <a className={logo} href="/">
-          Linaria
+        <a className={logoLink} href="/">
+          <img className={logo} src="/images/linaria-logo.svg" alt="Alt" />
```
Comment (zamotany) [inline website/src/components/Header.js]: `alt` attribute should be `Linaria logo`

File: website/src/components/Hero.js [hunk 2]
```diff
@@ -10,66 +10,158 @@ export default function Hero() {
...
 }
 
 const hero = css`
-  background: #fffaf6;
-  padding: 70px 0;
+  background: rgba(231, 56, 39, 1);
+  background: linear-gradient(
+    135deg,
+    rgba(231, 56, 39, 1) 0%,
+    rgba(154, 59, 165, 1) 0%,
```
Comment (zamotany) [inline website/src/components/Hero.js]: Is this `linear-gradient` correct? There're 2 colours at the same point: `rgba(231, 56, 39, 1) 0%,` and `rgba(154, 59, 165, 1) 0%,`

Comment (Trancever) [reply website/src/components/Hero.js]: Probably not, I wanted to create diagonal gradient and mess up a little bit. Will fix it.

**Comment by zamotany** [comment]
Also few things:
* run `yarn test:integration -u` to update website screenshot so the test won't fail
* run `yarn add-contibutor` and add yourself as a contibutor

File: website/src/components/Hero.js [hunk 3]
```diff
@@ -10,66 +10,153 @@ export default function Hero() {
...
+            <h1 className={heading}>Zero-Runtime CSS in JS</h1>
+            <p className={description}>
+              Linaria lets you write CSS in your JS with no runtime overhead
+            </p>
+            <a href="#get-started" className={button}>
+              Get Started
+            </a>
+          </div>
+          <div className={rightColumn}>
+            <img className={figureImage} alt="" src="/images/image-4.png" />
```
Comment (thymikee) [inline website/src/components/Hero.js]: alt="Linaria code sample"

File: website/src/components/Hero.js [hunk 4]
```diff
@@ -10,66 +10,153 @@ export default function Hero() {
...
+          </div>
+        </div>
       </Container>
+      <div className={wrapper}>
+        <div className={logoMarkContainer}>
+          <img className={logoMark} src="/images/linaria-logomark.svg" alt="" />
```
Comment (thymikee) [inline website/src/components/Hero.js]: alt="Linara logo"

File: website/src/components/Hero.js [hunk 5]
```diff
@@ -10,66 +10,153 @@ export default function Hero() {
...
+        </div>
+      </div>
     </div>
   );
 }
 
 const hero = css`
-  background: #fffaf6;
-  padding: 70px 0;
+  background: rgba(231, 56, 39, 1);
```
Comment (thymikee) [inline website/src/components/Hero.js]: Looks like no transparency needed?

File: website/src/components/Hero.js [hunk 6]
```diff
@@ -10,66 +10,153 @@ export default function Hero() {
...
-  border-radius: 3px;
+  border: 2px solid ${theme.white};
+  border-radius: 30px;
   cursor: pointer;
   transition: color 0.2s, background 0.2s;
 
   &:hover {
-    color: #fff;
-    background: ${theme.primary};
+    color: rgba(210, 53, 109, 1);
```
Comment (thymikee) [inline website/src/components/Hero.js]: No transparency here

File: website/src/components/Hero.js (hunk 5 above)
Comment (Trancever) [reply website/src/components/Hero.js]: Yeah, that's my mistake, I'll also change rgba to HEX.

**Pull request merged**